adm/src/commands/mod.rs

```diff
@@ -15,6 +15,7 @@
  * ------------------------------------------------------------------------------
  */
 
+pub mod batch;
```

Comment:
This should be behind the experimental feature defined in the Cargo toml, as should the uses and the clap args in main.rs.

Reply:
Fixed